### PR TITLE
Image Controller Same Scope Id And Name Fix

### PR DESCRIPTION
**Pull request by WASandaruwan**
Image Controller Same Scope Id And Name Fix

### CODE REVIEW

File: src/OnlineSales/Controllers/ImagesController.cs [hunk 1]
```diff
@@ -50,20 +51,34 @@ public async Task<ActionResult> Post([FromForm] ImageCreateDto imageCreateDto)
...
-                Data = imageInBytes,
-                MimeType = incomingFileMimeType!,
-                ScopeUid = imageCreateDto.ScopeUid,
-                Extension = incomingFileExtension,
-            };
-
-            await apiDbContext.Images!.AddAsync(uploadedImage);
-            await apiDbContext.SaveChangesAsync();
+                Image? uploadedImage = scopeAndFileExists!.FirstOrDefault();
+                uploadedImage!.Data = imageInBytes;
```
Comment (peterliapin) [inline src/OnlineSales/Controllers/ImagesController.cs:61]: Hi @WASandaruwan, what about updating the Size?

File: src/OnlineSales/Controllers/ImagesController.cs [hunk 2]
```diff
@@ -50,20 +51,34 @@ public async Task<ActionResult> Post([FromForm] ImageCreateDto imageCreateDto)
...
 
+                apiDbContext.Images!.Update(uploadedImage);
+                await apiDbContext.SaveChangesAsync();
```
Comment (peterliapin) [inline src/OnlineSales/Controllers/ImagesController.cs:64]: @WASandaruwan, SaveChangesAsync happens twice here and on the line 79. Why not to move it outside of if statement?

File: src/OnlineSales/Controllers/ImagesController.cs [hunk 3]
```diff
@@ -5,6 +5,7 @@
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.StaticFiles;
 using Microsoft.EntityFrameworkCore;
+using Nest;
```
Comment (peterliapin) [inline src/OnlineSales/Controllers/ImagesController.cs:8]: @WASandaruwan, what is the point to have Nest library included here? I do not see that you do anything with Elasticserch in the code of ImageController.cs ...

**Review by peterliapin — CHANGES_REQUESTED** [review]

File: src/OnlineSales/Controllers/ImagesController.cs (hunk 1 above)
Comment (WASandaruwan) [reply src/OnlineSales/Controllers/ImagesController.cs:60]: Size Added

**Pull request merged**